packages/Chem/src/package.ts

```diff
@@ -407,8 +407,11 @@ export function diversitySearchTopMenu(): void {
 //name: SearchSubstructureEditor
 //tags: editor
 //input: funccall call
-export function SearchSubstructureEditor(call: DG.FuncCall) {
-  
+export function searchSubstructureEditor(call: DG.FuncCall) {
```

Comment:
Why do we need this function? What does it do, is anyone using it?

Reply:
This is used by SubstructureSearchTopMenu function. I used this custom editor because in case we have only one molecule column we do not want function editor to open. It should automatically grab this column and open sketcher. If I'm not mistaken, in case we use just function with input molecule column parameter the input dialog will always be opened before running the fucntion. If it is ok, I will remove function editor.